src/scripts/interface/dialogManager.ts

```
@@ -22,6 +23,25 @@ export class DialogManager {
     }
   }
 
+  public static openEndDialog(text: string): Promise<boolean> {
+    this.endDialog.children[0].textContent = text;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)